sdk/storage/azure-storage-blob-batch/src/samples/java/com/azure/storage/blob/batch/BlobBatchJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,139 @@
+            .setLeaseAccessConditions(new LeaseAccessConditions().setLeaseId("{lease ID}"));
+
+        Response<Void> deleteResponse = batch.delete("{container name}", "{blob name}",
+            DeleteSnapshotsOptionType.INCLUDE, blobAccessConditions);
+        // END: com.azure.storage.blob.BlobBatch.delete#String-String-DeleteSnapshotOptionsType-BlobAccessConditions
+    }
+
+    /**
+     * Code snippet for {@link BlobBatch#delete(String, DeleteSnapshotsOptionType, BlobAccessConditions)}
+     */
+    public void addDeleteOperationWithUrl() {
+        // BEGIN: com.azure.storage.blob.BlobBatch.delete#String-DeleteSnapshotOptionsType-BlobAccessConditions
+        BlobAccessConditions blobAccessConditions = new BlobAccessConditions()
+            .setLeaseAccessConditions(new LeaseAccessConditions().setLeaseId("{lease ID}"));
+
+        Response<Void> deleteResponse = batch.delete("{url of blob}", DeleteSnapshotsOptionType.INCLUDE,
+            blobAccessConditions);
+        // END: com.azure.storage.blob.BlobBatch.delete#String-DeleteSnapshotOptionsType-BlobAccessConditions
+    }
+
+    /**
+     * Code snippet for {@link BlobBatch#setTier(String, String, AccessTier)}
+     */
+    public void addSimpleSetTierWithNames() {
+        // BEGIN: com.azure.storage.blob.BlobBatch.setTier#String-String-AccessTier
+        Response<Void> setTierResponse = batch.setTier("{container name}", "{blob name}", AccessTier.HOT);
+        // END: com.azure.storage.blob.BlobBatch.setTier#String-String-AccessTier
+    }
+
+    /**
+     * Code snippet for {@link BlobBatch#setTier(String, AccessTier)}
+     */
+    public void addSimpleSetTierWithUrl() {
+        // BEGIN: com.azure.storage.blob.BlobBatch.setTier#String-AccessTier
+        Response<Void> setTierResponse = batch.setTier("{url of blob}", AccessTier.HOT);
+        // END: com.azure.storage.blob.BlobBatch.setTier#String-AccessTier
+    }
+
+    /**
+     * Code snippet for {@link BlobBatch#setTier(String, String, AccessTier, LeaseAccessConditions)}
+     */
+    public void addSetTierWithNames() {
+        // BEGIN: com.azure.storage.blob.BlobBatch.setTier#String-String-AccessTier-LeaseAccessConditions
+        Response<Void> setTierResponse = batch.setTier("{container name}", "{blob name}", AccessTier.HOT,
+            new LeaseAccessConditions().setLeaseId("{lease ID}"));
+        // END: com.azure.storage.blob.BlobBatch.setTier#String-String-AccessTier-LeaseAccessConditions
+    }
+
+    /**
+     * Code snippet for {@link BlobBatch#setTier(String, AccessTier, LeaseAccessConditions)}
+     */
+    public void addSetTierWithUrl() {
+        // BEGIN: com.azure.storage.blob.BlobBatch.setTier#String-AccessTier-LeaseAccessConditions
+        Response<Void> setTierResponse = batch.setTier("{url of blob}", AccessTier.HOT,
+            new LeaseAccessConditions().setLeaseId("{lease ID}"));
+        // END: com.azure.storage.blob.BlobBatch.setTier#String-AccessTier-LeaseAccessConditions
+    }
+}
```

Comment:
Can we also have some sort of samples file that shows an end-to-end sample of how to construct a batch, submit it, and then process the responses once they have been filled?

Reply:
I'll add a non-Javadoc snippets sample file that will go through an end-to-end scenario.